src/styles/global.css

```diff
@@ -20,27 +20,88 @@ body {
   font-size: 17.6px;
```

Comment:
Wtf is this value?

Reply:
Need to be in rem as well

Reply:
17.6px. Why does it need to be in REM. I disagree